src/redux/attachments/types.ts

```diff
@@ -1,5 +1,10 @@
+import { ICard } from '../cards/types'
+
 export type IAttachment = {
     id: number,
-    type: string, 
-    URL: string
+    URL: string,
+    pos: number,
+    name: string,
+    date: Date,
+    card: ICard
```

Comment:
A attachment has a card, that has many attachment, and each will have a card, that has many attachments... You get the idea, this is wrong.

Reply:
I changed it with the id.